Shoko.Server/Repositories/Direct/RenameScriptRepository.cs

```diff
@@ -40,5 +40,17 @@ public RenameScript GetByName(string scriptName)
                 .List<RenameScript>().FirstOrDefault();
             return cr;
         }
+
+        public List<RenameScript> GetRenamerType(string renamerType)
+        {
+            if (string.IsNullOrEmpty(renamerType)) return new();
```

Comment:
Why are we returning a new list if there is no type?

Reply:
To avoid checking for and handling null when expecting a list.